docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/content/015_article_cancer_genomics_research.content.yml

```diff
@@ -104,7 +104,7 @@
       type: "body_section"
       field_body_section_heading:
         - format: "simple"
-          value: "Why Genomics Research Is Critical to Progress against Cancer"
+          value: "Why Genomics Research Is &quot;Critical&quot; to Progress  &amp; against &lt;Cancer&gt;"
       field_body_section_content:
```

Comment:
Please add a case for some named entity with upper case vs lower case so we can catch any regressions.  Please also add one or more examples for numeric entities, e.g. `&#x00397;&#xeB;&#x139;&#x13A;&#x1d560;`

Reply:
Added some examples as suggested.